### PR TITLE
feat(datastore): support custom object type

### PR DESCRIPTION
**Pull request by xuchuan**
## Modules
- [ ] UI
- [X] Controller
- [ ] Agent
- [] Client
- [X] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [X] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/controller/src/test/java/ai/starwhale/mlops/datastore/DataStoreTest.java [hunk 1]
```diff
@@ -182,6 +182,50 @@ public void testQuery() {
...
+                        Map.of("k", "5", "x:link/url", "http://test.com/1.jpg", "x:link/mime_type", "image/jpeg"),
+                        Map.of("k", "6", "x:link/url", "http://test.com/2.png", "x:link/mime_type", "image/png"))));
+
+        recordList = this.dataStore.query(DataStoreQueryRequest.builder()
+                .tableName("t1")
+                .columns(Map.of("x", "y", "x:link/url", "url"))
+                .build());
+        assertThat("object type alias",
+                recordList.getColumnTypeMap(),
+                is(Map.of("url", ColumnType.STRING, "y:link/mime_type", ColumnType.STRING)));
```
Comment (jialeicui) [inline server/controller/src/test/java/ai/starwhale/mlops/datastore/DataStoreTest.java:219]: It seems that `columnAlias` makes no guarantees as to the order of the map.

Comment (xuchuan) [reply server/controller/src/test/java/ai/starwhale/mlops/datastore/DataStoreTest.java:219]: Yes, maps in the results are always unordered.

**Comment by jialeicui** [comment]
LGTM

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**